lib/BuildSystem/BuildSystemFrontend.cpp

```diff
@@ -232,12 +232,21 @@ class BuildSystemFrontendExecutionQueueDelegate
 };
 
 struct BuildSystemFrontendDelegateImpl {
+
+  /// The status of delegate.
+  enum class Status {
+    Uninitialized = 0,
```

Comment:
This term is a little confusing given `initialize` -- the system can be "initialized" w.r.t. `initialize()`, but the Status be `Uninitialized` which is a little odd.